apps/webstore/src/app/shared/components/modals/change-password-modal/change-password-modal.component.html

```diff
@@ -0,0 +1,113 @@
+<div class="modal-container">
+    <div class="modal-content">
+        <div class="modal-header-secondary">
+            <img
+                src="/assets/icons/shield-icon.svg"
+                class="modal-header-icon"
+            />
+        </div>
+        <form
+            (ngSubmit)="changePassword()"
+            name="changePasswordForm"
+            [formGroup]="changePasswordForm"
+        >
+            <div class="modal-body-secondary">
+                <div class="title">
+                    <h3 i18n="@@change-password">Change password</h3>
+                    <!-- new password -->
+                    <div class="tc-input-group field">
+                        <input
+                            type="text"
+                            formControlName="password"
+                            class="tc-input-field"
+                            placeholder="E-mail"
+                            name="password"
+                            id="password"
+                            required
+                        />
+                        <label
+                            for="password"
+                            class="tc-input-label"
+                            i18n="@@new-password"
+                            >New password</label
+                        >
+                    </div>
+                    <div class="info-message" i18n="@@info-password">
+                        Password must contain: min 9 characters, at least one
```

Comment:
The minimum password length is 6 characters
```suggestion
                        Password must contain: min 6 characters, at least one
```